temp-scripts/fix-all-linting.sh

```
@@ -0,0 +1,92 @@
+# Final check
+cd /Users/MAC/Documents/projects/caia
+npx lerna run lint --parallel 2>&1 | grep -c "Lifecycle script \`lint\` failed" | read failures
+
+if [ "$failures" == "0" ]; then
+  echo "✅ All packages pass linting!"
+else
+  echo "⚠️ Some packages still have linting issues. Running detailed check..."
+  
+  # Check each package
+  for pkg in "${UTILITIES[@]}"; do
+    cd "/Users/MAC/Documents/projects/caia/$pkg" 2>/dev/null
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)